staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -147,6 +149,8 @@ func (s *DelegatingAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 	if s.RemoteKubeConfigFileOptional {
 		optionalKubeConfigSentence = " This is optional. If empty, all token requests are considered to be anonymous and no client CA is looked up in the cluster."
 	}
+	fs.StringSliceVar(&s.APIAudiences, "api-audiences", s.APIAudiences, "Identifiers of the API. Authenticators will validate that tokens used against the API are bound to at least one of these audiences.")
```

Comment:
"Identifiers of the API" is pretty vague, but I'm struggling to come up with anything much better. Maybe "Identifiers of this apiserver."?

Reply:
Changed to "Identifiers of this API server. Authenticators will validate that tokens used against this API server are bound to at least one of these audiences."